### DISCUSSION TITLE
UC 2 Backend Done

### DISCUSSION THREAD

**Discussion by hueyee** [SparkSketch-Team/SparkSketch#42] [General]
I had to refactor the URLs for both image upload and view posts. @acsmaker-code You will have to refactor those two pages to accommodate.

**Comment by acsmaker-code** [comment]
What do you mean by refactor like did you just change the urls or did you change the backend

**Reply by hueyee** [reply]
I changed the URLs for the http requests for more clarity. For the feed, you are now given more data than just a link. Run the backend in Swagger UI to see
